### PR TITLE
MCS H5 Recording Extractor

### PR DESCRIPTION
**Pull request by p2jones**
https://github.com/SpikeInterface/spikeextractors/issues/235

### CODE REVIEW

File: spikeextractors/extractors/mcsh5recordingextractor/mcsh5recordingextractor.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        data_V = np.array(stream.get('ChannelData'),dtype=np.int)*self._convFact.astype(float)*(10.0**(self._exponent))
+
+        return data_V[channel_indices, start_frame:end_frame]
+
+    @staticmethod
+    def write_recording(recording, save_path):
+        # Not implemented
+        # An informative example is in BiocamRecordingExtractor
+
+        assert HAVE_MCSH5, "To use the MCSH5RecordingExtractor install h5py: \n\n pip install h5py\n\n"
```
Comment (alejoe91) [inline spikeextractors/extractors/mcsh5recordingextractor/mcsh5recordingextractor.py]: You can also just remove the method or `raise NotImplementedError`

File: spikeextractors/extractorlist.py [hunk 2]
```diff
@@ -32,7 +33,8 @@
     SpykingCircusRecordingExtractor,
     SpikeGLXRecordingExtractor,
     PhyRecordingExtractor,
-    MaxOneRecordingExtractor
+    MaxOneRecordingExtractor,
+	MCSH5RecordingExtractor
```
Comment (alejoe91) [inline spikeextractors/extractorlist.py]: can you fix the indentation?

**Comment by colehurwitz** [comment]
Ready to merge?

**Comment by p2jones** [comment]
Should docs be updated at the same time?

**Comment by colehurwitz** [comment]
Which docs are you referring to? I think we can merge now and fix any of that later.

**Comment by p2jones** [comment]
I mean spikeextractors/docs/available_extractors.rst 
But merging now is fine with me!

**Comment by colehurwitz** [comment]
Is mea_pitch needed? It isn't used anywhere in the extractor.

**Comment by p2jones** [comment]
mea_pitch can be removed.

**Pull request merged**